miraclelinux/meta-debian-extended
# Add nginx package.

# Purpose of pull request

nginx 

This pr adds FOOBAR package based on meta-openembedded's meta-webserver/recipes-httpd/nginx/nginx_1.14.2.bb

# Test
## How to test

Add nginx and rebuild the image.
```
IMAGE_INSTALL_append = " nginx"
```
It will start automatically, so make sure the server is running.

## Test result

Make sure that nginx is running.
```
root@qemuarm64:~# ps | grep nginx
 1653 root      7380 S    nginx: master process /usr/sbin/nginx
 1655 www       7760 S    nginx: worker process
```
Connect using a web browser on the host side.
http://192.168.7.2
Addresses will vary depending on the situation.

If you see `Welcome to nginx!`, you have succeeded.

**Caution**
An error will occur in environments where IPv6 is not available.
In this case, comment out the "listen [::]:80 default_server;" item in "/etc/nginx/sites-available/default_server".
